data/data-pipeline/data_pipeline/etl/score/config/datasets.yml

```diff
@@ -117,6 +117,34 @@ datasets:
         field_type: float
         include_in_downloadable_files: true
         include_in_tiles: true
+  - long_name: "Formerly Used Defense Sites"
+    short_name: "FUDS"
+    module_name: "us_army_fuds"
+    load_fields:
+      - short_name: "eligibile_count"
```

Comment:
Perhaps just `fuds_count`? I think that for the shapefile and many dataset reasons below, might be good to include this! (unless my original hope of `[module_name]__[short_name]` is the plan :()